src/angular-core/styles/typography/_table.scss

```diff
@@ -1,3 +1,5 @@
+@import '../common';
```

Comment:
Remove this import. The imports were removed for cleaner bundling. The other files are implicitely imported during bundling.